### ISSUE TITLE
Arxiv paper doesn't have alt texts

### ISSUE THREAD

**Issue by domoritz** [allenai/hci-alt-texts#4]
Could you link to another version of the paper that has alt texts for the images? For example https://dl.acm.org/doi/pdf/10.1145/3517428.3544796.
